Core/include/Acts/Geometry/detail/PortalHelper.hpp

```diff
@@ -73,6 +73,60 @@ defaultPortalAndSubPortalGenerator() {
   return pGenerator;
 }
 
+/// Definition of a portal replacement when building proto containers
+/// It consists of the new portal, the index, the direction, the parameters
+/// gathered from the sub volumes, the binning description
+using PortalReplacement =
+    std::tuple<std::shared_ptr<Experimental::Portal>, unsigned int,
+               NavigationDirection, std::vector<ActsScalar>, BinningValue>;
```

Comment:
Could this be an extra `struct` with names? This has 5 elements and might start being confusing somewhat.

Note you can still destructure this like a tuple if it's plain struct like you do below, I believe.

```cpp
for (auto& [p, i, dir, boundaries, binning] : pReplacements) {}
```

Reply:
Ok, will look into this, if i'ts deconstructable it's fine.

Reply:
Will move this comment to the `Detector` PR where these classes are touched again, and will try it.